bitrise.yml

```diff
@@ -133,6 +133,7 @@ workflows:
           inputs:
             - content: |-
                 cd example-app/ios
+                pod update hermes-engine --no-repo-update
```

Comment:
Hey @bric-stripe, trouble you again😬. I'm working on upgrading expo and met a issue of building on ios. Base on the latest expo, I also upgraded the dependencies as expo requests, including react-native with the hermes-engine.  first error came as
<img width="1175" alt="image" src="https://github.com/user-attachments/assets/aba46de1-1a6f-4338-ae30-10db4d179bb1">
then I added this line 'pod update hermes-engine --no-repo-update' to fix. but it cause current issue as 
<img width="1196" alt="image" src="https://github.com/user-attachments/assets/c6caf60d-613e-4860-9d32-ceb6d817a658">
I've tried several way to try to fix it, like changing Podfile to set `'use_modular_headers!'`  or use :modular_headers => true for  'DoubleConversion' , but there is no lucky. Have you seen such a error before and had experiences to fix it ? Please take a look when you get a chance, thanks~.

Reply:
update: I found a workaround that set `use_frameworks! :linkage => :static` in Podfile, and it works.